script.js

```diff
@@ -1,4 +1,160 @@
+/*Create a version of the Beat That dice game, where players create the largest number they can by concatenating random dice roll digits:
+
+There are 2 players and players take turns.
+When a player clicks Submit, the game rolls 2 dice and shows the dice rolls, for example 3 and 6.
+The player picks the order of the dice they want. For example, if they wanted the number 63, they would specify that the 2nd dice goes first. You can choose how the player specifies dice order.
+After both players have rolled and chosen dice order, the player with the higher combined number wins.*/
```

Comment:
Good job using pseudocode to plan what needs to be coded before starting to code!